connection_manager.go

```diff
@@ -379,7 +577,7 @@ func (n *connectionManager) makeTrafficDecision(localIndex uint32, now time.Time
 			// This is similar to the old punchy behavior with a slight optimization.
 			// We aren't receiving traffic but we are sending it, punch on all known
 			// ips in case we need to re-prime NAT state
-			n.sendPunch(hostinfo)
+			//n.sendPunch(hostinfo)
```

Comment:
imo we should change this punch behavior separately

Reply:
Agree, the default UDP timeout in many NATs is going to be 180 seconds. If we remove punching then the maximum we can somewhat safely assume an idle tunnel can idle for is 3 minutes.

Reply:
We should modify this entire function to not punch to lighthouses, which means that we trend toward idle/no punches when there are no tunnels besides lighthouses.